### PR TITLE
adds pinterest - today page and single photo

### PR DESCRIPTION
**Pull request by samejima-san**
## Description

> I added support to Pinterest, get the today page as well as a extracting images from the page without creating an account.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [x] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

> Uploading a screenshot illustrating the approach you developed for validation is required.

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

**Comment by nikhil25803** [comment]
Go ahead @samejima-san 
**Note**
+ Create a separate module for this, as per the folder and project structure
+ Do not use the `selenium web driver as it is incompatible with all devices and cloud platforms.
+ Before making any changes, please check whether the module you want to add exists. If yes, then you can just add your functionality as a method only make a separate module and class for it.

File: src/scrape_up/pinterest/pinterest.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        | ---------------------- | ------------------------------------------------------------------ |
+        | `.get_today()`         | Returns the list of todays topics                                  |
+        | `.get_photo(your url)` | Returns the link to the image (so you dont need an account)        |
+    """
+    def __init__(self):
+        pass
+
+    def get_today(self):
+        """
+            Class - `Pinterest`
```
Comment (nikhil25803) [inline src/scrape_up/pinterest/pinterest.py:21]: Format docstring, check modules like GitHub and IMDB and you'll get an idea about how we format docstring of each mehods.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by samejima-san** [comment]
@nikhil25803 no worries good luck with school! 
I can help maintain this project if you need another set of eyes on it.

**Comment by nikhil25803** [comment]
Sure @samejima-san, will let you know :))

**Pull request merged**